civihr_default_theme/assets/sass/partials/_page.scss

```diff
@@ -20,8 +20,12 @@
       overflow: visible !important;
     }
 
-    #bootstrap-theme .dropdown-menu[datepicker-popup-wrap] {
-      width: 316px;
+    #bootstrap-theme {
+      display: inline-block;
```

Comment:
Because the `#bootstrap-theme` element is added between the input and the icon, so without the `display: inline-block;` the layout break.

![image](https://cloud.githubusercontent.com/assets/1280255/16735430/62fed56c-475f-11e6-8042-f58bd0817eb2.png)